CMakeLists.txt

```diff
@@ -91,6 +91,12 @@ if (NOT RAJA_LOADED)
   if (ENABLE_OPENMP)
     set (raja_depends
       openmp)
+
+  if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Intel")
+    message(WARNING "RAJA::simd_exec not supported ")
```

Comment:
Can you make the message say something like "simd_exec not supported when OpenMP is enabled with the Intel compiler"